pilot/pkg/model/extensions.go

```diff
@@ -36,19 +37,26 @@ const (
 	defaultRuntime = "envoy.wasm.runtime.v8"
 	fileScheme     = "file"
 	ociScheme      = "oci"
+
+	// name of environment variable at Wasm VM, which will carry the Wasm image pull secret.
+	WasmSecretEnv = "ISTIO_META_WASM_IMAGE_PULL_SECRET"
```

Comment:
This leaks out through ecdsz to any authenticated user

Reply:
Make it redacted at ecdsz.